src/components/global/DocsCard/index.tsx

```diff
@@ -34,14 +34,16 @@ function DocsCard(props: Props): JSX.Element {
         )}
         {props.ionicon && <ion-icon name={props.ionicon} className="Card-ionicon"></ion-icon>}
         {props.iconset && (
```

Comment:
I searched the repository and `iconset` was not used anywhere so I stole this property to make my card that shows icon + icon:

<img width="696" height="424" alt="Image" src="https://github.com/user-attachments/assets/91c35d0c-fa61-487b-8a66-40d8552935ed" />

Reply:
Woot to making it useful!!